DEVELOPING_OPAL/validate/src/test/java/org/opalj/fpcf/fixtures/alias/ShapeTest.java

```diff
@@ -0,0 +1,229 @@
+package org.opalj.fpcf.fixtures.alias;
+
+import org.opalj.fpcf.properties.alias.Alias;
+import org.opalj.fpcf.properties.alias.MayAlias;
+import org.opalj.fpcf.properties.alias.MustAlias;
+import org.opalj.fpcf.properties.alias.NoAlias;
+
+public class ShapeTest {
```

Comment:
Wouldn't call these things "Test", they aren't tests, they are fixtures used by tests. Naming convention in OPAL here is just to name them after what they contain.